### DISCUSSION TITLE
Show and Tell, a work in progress

### DISCUSSION THREAD

**Discussion by bobcgausa** [shakiba/planck.js#198] [Show and tell]
https://sites.google.com/site/professorcookga/planck-box2d-physics-for-javascript-p5

p2.js is a library for Planck that was designed to work with a pixel interface for p5js.org

I wrapped Body completely, but am undecided on whether to do the same for fixtures.

**Comment by shakiba** [comment]
Looks great, thanks for sharing!

Just one small note, there is another physics library called p2.js.
